config/scheduler/settings.ini

```diff
@@ -98,7 +98,12 @@ pa.scheduler.core.automaticremovejobdelay=0
 # This housekeeping feature can be replaced by a stored procedure
 # that runs at the desired period of time (e.g. non-business hours)
 # Such an example is available in samples/scripts/database/postgres/
-pa.scheduler.job.removeFromDataBase=false
+pa.scheduler.job.removeFromDataBase=true
+
+# This cron expression determines the housekeeping call frequency.
+# Default value is 1 minute: this will invoke the housekeeping mechanism
+# to remove any job that are set to be removed and has their scheduled time for removal reached.
+pa.scheduler.core.automaticremovejobexpression=* * * * *
```

Comment:
I think the default cron expression should be a bit higher (10 minutes, 1 hour) ?